projects/lwan/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gcr.io/oss-fuzz-base/base-builder
```

Comment:
Could you please add the copyrights to this file and build.sh? They are usually created automatically by `infra/helper.py` script, or you can copy it from any other project files.